.taprc

```diff
@@ -1,2 +1,3 @@
+disable-coverage: true
```

Comment:
It was actually turned off. It is on by default in tap 18

Reply:
Ok, here we should remove it because originally the coverage was 100%

We can see here that the istanbul ignore is no longer working:

https://github.com/fastify/fastify-jwt/blob/77721ccfc9f0ccf1daf477a768eb42827ba48a23/jwt.js#L217-L226

So you can remove the istanbul-ignore and add something like:

```js
catch (error) {
      // Ignoring the else branch because it's not possible to test it,
      // it's just a safeguard for future changes in the fast-jwt library
      if (error.code === TokenError.codes.malformed) {
        throw new AuthorizationTokenInvalidError(error.message)
      } else if (error.code === TokenError.codes.invalidType) {
        throw new AuthorizationTokenInvalidError(error.message)
      } /* c8 ignore start */ else {
        throw error
      }
      /* c8 ignore stop */
    }
```